packages/import-notation/README.md

```diff
@@ -80,6 +78,23 @@ Parameter | Type     | Description
 Forms a string from [BEM entities]. Be aware to merge only one type of entities.
 The array should contains one block or one elem and optionally it's modifiers.
 
+### expand(str, ctx)
+
+Parameter | Type     | Description
+----------|----------|--------------------------------------------------------
+`str`     | `string` | BEM import notation check [notation section](#notation)
+`ctx`     | `object` | BEM entity name representation
+
+Expand notation string to full form by context BEM entity.
+
+Example:
+
+```js
+var notation = parse('e:text', { block: 'button' });
```

Comment:
expand?

Reply:
Also, I didn't understand the difference from parse method.
Actually, I understood but it was not so obvious 
add examples of the distinction between them, please

Reply:
Are you sure we need 2 methods? ;-)

Reply:
Sure! `expand` gives us string and it is fast/lightweight

Reply:
Are you sure `expand` is the most viable name for this functionality?

I'd suggest:
- `fullfill`
- `fillup`
- `sate`